application/templates/finished.html

```diff
@@ -0,0 +1,35 @@
+{% extends "base/lr_base_layout.html" %}
```

Comment:
The indentation is a bit messed up because of the mixture of tabs and spaces as can be seen in the image attached. Use either tabs or spaces, dont really mind which you use but try to be consistent with both yourself and other similar files

<img width="177" alt="screen shot 2016-01-29 at 14 40 55" src="https://cloud.githubusercontent.com/assets/3599604/12678729/2000c434-c698-11e5-8195-a598ce60e4c9.png">